test/functional/admin/documents_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'minitest/stub_any_instance'
+
+module Admin
+  class DocumentsControllerTest < ActionController::TestCase
```

Comment:
So these tests are just testing responses. They're not testing that what we want to happen has happened. Let's discuss at our meeting today, but ahead of that meeting please think about, for each of these methods, what we want the method to do and how we could test them.